performance/client/src/__tests__/primary/ExplorerAPIs/get-job-list.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright IBM Corporation 2020
+ */
+
+import WrkTestCase from "../../../testcase/wrk";
+import { getBasicAuthorizationHeader } from "../../../utils";
+
+class ExplorerApiJobListTest extends WrkTestCase {
+  fetchZoweVersions = true;
+
+  name = "Test explorer data sets api endpoint /api/v2/jobs";
+  endpoint = '/api/v2/jobs';
```

Comment:
We may need some filters for this endpoint. Without filters, it may try to pull all jobs and will be very slow.

Reply:
I've modified this test to filter on `SDSF`* prefix and `ACTIVE` job status, which should limit the scope of what is pulled with this test case while still being usable between different systems.

Reply:
I made one additional change here to have this test case pull a list of the SDSF job only for increased consistency and to filter with any owner to ensure the SDSF job will show up in any case.